common/test/acceptance/tests/lms/test_lms_course_home.py

```diff
@@ -164,3 +167,77 @@ def test_course_search_a11y(self):
             ]
         })
         course_search_results_page.a11y_audit.check_for_accessibility_errors()
+
+
+class CourseOutlineTest(UniqueCourseTest):
+    """
+    Test Suite to verify the course outline page on the LMS.
+    """
+    USERNAME = "STUDENT_TESTER"
+    EMAIL = "student101@example.com"
+
+    def setUp(self):
+        """
+        Initialize pages and install a course fixture.
+        """
+        super(CourseOutlineTest, self).setUp()
+
+        self.course_home_page = CourseHomePage(self.browser, self.course_id)
+        # Install a course with sections and problems
+        self.course_fix = CourseFixture(
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run'],
+            self.course_info['display_name'],
+            start_date=datetime.now() + timedelta(days=-10),
+            end_date=datetime.now() + timedelta(days=10)
+        )
+
+        self.course_fix.add_children(
+            XBlockFixtureDesc('chapter', 'Test Section').add_children(
+                XBlockFixtureDesc('sequential', 'Test Subsection', metadata={
+                    'due': (datetime.now() + timedelta(days=-20)).isoformat(),
+                    'format': 'Homework'
+                }).add_children(
+                    XBlockFixtureDesc('problem', 'Test Problem', data=load_data_str('multiple_choice.xml')),
+                )
+            ),
+        ).install()
+        # Auto-auth register for the course.
+        auto_auth(self.browser, self.USERNAME, self.EMAIL, False, self.course_id)
+
+    def change_course_pacing_to_self_paced(self):
+        """
+        Change the course pacing from Instructor Paced to Self-paced course
+        for a live course.
+        """
+        # Change start date to future to be able to change pacing
+        self.course_fix.add_course_details({'start_date': (datetime.now() + timedelta(days=5))})
+        self.course_fix.configure_course()
+        # Change the pacing to self_paced
+        self.course_fix.add_course_details({'self_paced': True})
```

Comment:
Why can't you add all the course details one time and call `configure_course`? I can you are doing in 3 steps and after each you are calling configure_course.

Reply:
`configure_course` does an API call to the studio API to update the course. If I don't call configure_course, the changes will not have any effect.

Reply:
This was also the reason I had to add comments to explain the 3 calls happening.

Reply:
Why can't this be done in single call?

Reply:
Code explaining comments are not appreciated but on rare cases.

Reply:
Since it is a live course, so I can't change the pacing unless the start date is in the future. So, the first part will change the start date into a future date to allow us to change pacing. Next call will be to change the pacing itself. and finally, the last call is setting the start date in the past so that learner can see the course and visit the course dashboard.

Reply:
Ok, is there a reason this course can't have a start date in the future in the first place? 

Is the course outline is accessible If the course start date is in the future?

Reply:
No. For a learner, the error will say that the course you are looking for doesn't start until that future date.

Reply:
Got it. 
Just update the comments in the method rest is all good :)